src/main/java/net/sf/mzmine/modules/peaklistmethods/io/spectraldbsubmit/sorting/ScanSorter.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2006-2018 The MZmine 2 Development Team
+ * 
+ * This file is part of MZmine 2.
+ * 
+ * MZmine 2 is free software; you can redistribute it and/or modify it under the terms of the GNU
+ * General Public License as published by the Free Software Foundation; either version 2 of the
+ * License, or (at your option) any later version.
+ * 
+ * MZmine 2 is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public License along with MZmine 2; if not,
+ * write to the Free Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301
+ * USA
+ */
+
+package net.sf.mzmine.modules.peaklistmethods.io.spectraldbsubmit.sorting;
+
+import java.util.Comparator;
+import net.sf.mzmine.datamodel.MassList;
+import net.sf.mzmine.datamodel.Scan;
+import net.sf.mzmine.util.ScanUtils;
+import net.sf.mzmine.util.exceptions.MissingMassListException;
+
+public class ScanSorter implements Comparator<Scan> {

```

Comment:
This class can be of general use, so you should move it to the net.sf.mzmine.util package

Reply:
done